src/char/hammerdin.py

```diff
@@ -113,19 +119,30 @@ def kill_council(self) -> bool:
             wait(0.05, 0.15)
         # Check out the node screenshot in assets/templates/trav/nodes to see where each node is at
         atk_len = self._char_config["atk_len_trav"]
-        # Go inside and hammer a bit
-        self._pather.traverse_nodes([228, 229], self, time_out=2.5, force_tp=True)
-        self._cast_hammers(atk_len)
         # Move a bit back and another round
         self._move_and_attack((40, 20), atk_len)
         # Here we have two different attack sequences depending if tele is available or not
         if self.can_teleport():
+            # Go inside and hammer a bit
+            self._pather.traverse_nodes([228, 229], self, time_out=2.5, force_tp=True)
+            self._cast_hammers(atk_len)
+            self._move_and_attack((40, 20), atk_len)
             # Back to center stairs and more hammers
             self._pather.traverse_nodes([226], self, time_out=2.5, force_tp=True)
             self._cast_hammers(atk_len)
             # move a bit to the top
             self._move_and_attack((65, -30), atk_len)
+            self._move_and_attack((-125, -373), atk_len)
         else:
+            # Go to center stairs a bit
+            self._pather.traverse_nodes([226], self, time_out=2.5, force_tp=True, force_move=True)
```

Comment:
Isnt that the node we are at anyway after the traverse to "safe_dist"?

Reply:
This will start hammer on the outside for this node before go in